src/Resources/config/services.yml

```diff
@@ -0,0 +1,7 @@
+services:
+  cca.events_cron.cron_dispatcher:
+    class: ContaoCommunityAlliance\Contao\Events\Cron\CronDispatcher
+    arguments:
+      - '@event_dispatcher'
+      - '@cca.translator.contao_translator'
```

Comment:
I do not see the translator being used at all, maybe this argument (which is also not used) should be removed? :smile_cat:

Reply:
Crazy. Done in d1db9f7f533370155c9ca806aaab922bb4bd2ab4.